### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by Orebeer** [mark-hahn/markdown-scroll-sync#41]
I've read all comments on this feed and tried to emulate every possible environment by disabling all other possible third-party tie-in contingencies. It still does not work so I've given up. 

Mark if you can get this to work, it's a big deal for the Markdown community on Atom. It's frankly surprising that Atom hasn't focused on this feature themselves. Someone eventually will as it's too important of a feature to ignore. 

Mark one suggestion would be for you to document your own working environment as a baseline. If we could start from your working instance and then collectively enable third-party tools until we identify the culprit, it would make it much easier for you to remediate.

**Comment by mark-hahn** [comment]
> Mark if you can get this to work, it's a big deal for the Markdown
community

I apologize.  I haven't had any problem and I've been too lazy to research
the problem.  I will take another look.

> surprised they haven't focused on it themselves but someone will
eventually

It doesn't take a pull request to help.  Just finding the source of the
problem would enable an easy fix.  Surely there are some programmers using
Atom.  (grin).

The normal way would be for me to ask others to do specific tests.  I will
try to do this soon.

I have a general problem that I've created too many packages (25?)  and
they require more support than expected.  The reason I created so many
packages is that I want them for my use.  So I've only been fixing problems
when they happen to me.  I do understand that I owe support to users when I
publish a package. I'm guessing that about a third of my packages have
outstanding issues.
